lib/puppet/provider/user/useradd.rb

```diff
@@ -9,8 +9,12 @@
     install Ruby's shadow password library (often known as `ruby-libshadow`)
     if you wish to manage user passwords."
 
+  # These commands have special names for use with the base ObjectAdd provider
   commands :add => "useradd", :delete => "userdel", :modify => "usermod", :password => "chage"
 
+  # This is used in our local implementation and has no special name
+  commands :gpasswd => "gpasswd"
```

Comment:
And now that I write that I don't understand why we're switching from usermod to gpasswd as part of this ticket?